packages/slice-machine/src/legacy/components/AppLayout/index.tsx

```diff
@@ -18,7 +26,31 @@ export const AppLayout: FC<PropsWithChildren> = ({
   ...otherProps
 }) => {
   return (
-    <ErrorBoundary>
+    <ErrorBoundary
+      renderError={(error) => {
```

Comment:
i guess there's no way to recover from this error?

Reply:
Nope, these errors need support or knowledge from the user to fix it, but SM should not be used in the meantime.

Note: maybe we will improve that if we detect some recovery paths for some errors.